presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,54 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Fix a null pointer exception during the stats collection phase in query planning.
+* Fix an issue where outer joins without equality predicates would sometimes run on a single node
+  when ``join_distribution_type`` was set to ``AUTOMATIC``.
+* Fix equality inference for ``VARCHAR`` columns.  This improves performance for queries that have
```

Comment:
```suggestion
* Improve performance for queries with constant VARCHAR predicates in `WHERE` or `JOIN` clause
```

Reply:
That's not quite right, but I'm not sure of a good way to say it.  the improvement is for queries like
`SELECT * FROM t1 JOIN t2 ON t1.my_varchar = t2.my_varchar WHERE t1.my_varchar = 'hello'`.  It lets 'hello' get pushed down to the t2 table also.

Reply:
maybe 
```
* Improve performance for queries with constant ``VARCHAR`` predicates on join columns
```

Reply:
I like it!